core/src/test/java/com/linecorp/armeria/server/HttpServerAutoRedirectTest.java

```diff
@@ -61,9 +61,6 @@ protected void configure(ServerBuilder sb) throws Exception {
             // returning `202 Accepted`.
             sb.service("/f/", service);
             sb.routeDecorator().pathPrefix("/f/").build((delegate, ctx, req) -> HttpResponse.of(202));
-
-            // This should never be invoked in this test.
-            sb.serviceUnder("/", service);
```

Comment:
@trustin I removed this line because of the following test:
```
            // `GET /e` should be redirected to `/e/`.
            // `DELETE /e` should NOT be redirected to `/e/`.
            sb.route().get("/e/").build(service);
```
Because my fix made `/e/`'s fallback route support HTTP GET only. So the request would be routed the '/' service. Please let me know if I'm wrong. 😄